incubating/wrappers/s2i/java-jni/test/model-template-app/pom.xml

```diff
@@ -0,0 +1,83 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.seldon.example</groupId>
+  <artifactId>seldon-core-model-template</artifactId>
+  <packaging>jar</packaging>
+  <version>0.3.0-SNAPSHOT</version>
+  <name>Seldon Core Model Template</name>
+  <url>http://maven.apache.org</url>
+  <properties>
+    <java.version>13</java.version>
+    <maven.compiler.source>13</maven.compiler.source>
+    <maven.compiler.target>13</maven.compiler.target>
+    <project.build.sourceEncoding>
+    UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>
+    UTF-8</project.reporting.outputEncoding>
+    <start-class>io.seldon.example.App</start-class>
+  </properties>
+  <build>
+    <finalName>${project.artifactId}-${project.version}</finalName>
+    <extensions>
+      <extension>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.4.1.Final</version>
+      </extension>
+    </extensions>
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.5.1</version>
+        <configuration>
+          <source>13</source>
+          <target>13</target>
+          <encoding>UTF-8</encoding>
+        </configuration>
+      </plugin>
+      <plugin>
+        <!-- Build a fat JAR with all dependencies -->
+        <artifactId>maven-assembly-plugin</artifactId>
+        <configuration>
+          <descriptorRefs>
+            <descriptorRef>jar-with-dependencies</descriptorRef>
+          </descriptorRefs>
+          <appendAssemblyId>false</appendAssemblyId>
+        </configuration>
+        <executions>
+          <execution>
+            <id>make-assembly</id>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+    <dependency>
+      <groupId>io.seldon.wrapper</groupId>
+      <artifactId>seldon-core-wrapper</artifactId>
+      <version>0.3.0-SNAPSHOT</version>
+    </dependency>
+  </dependencies>
+  <repositories>
+    <!-- Download seldon-core-wrapper:0.3.0-SNAPSHOT from snapshots repo. Remove once it's published. -->
```

Comment:
Will this have to stay when the PR is merged?

Reply:
Good spot! We should be good to publish `0.3.0` of the `seldon-core-wrapper` package. It's still backwards compatible (i.e. it's still got Spring, Tomcat and everything else embedded).